WordPress/Classes/Models/JetpackSiteRef.swift

```diff
@@ -31,4 +37,8 @@ struct JetpackSiteRef: Hashable, Codable {
         return lhs.siteID == rhs.siteID
             && lhs.username == rhs.username
```

Comment:
WDYT of adding checks for `hasBackup` and `hasPaidPlan` for `==(lhs: rhs:)` just to be safe?

Reply:
Done in 03b314dd274da2303f0c40d8379f7e6c418ab4bf